src/main/webapp/js/utils/table-cells/collection-cell.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { getEntityArrayByName } from '../../selectors';
+
+const CollectionCell = ({ entityArray, relatedField }) => {
+  const collectionCellValue = JSON.stringify(entityArray[0][relatedField]);
```

Comment:
I think you should create a selector to get the list of values.
In this component use `_.join(values, ', ')` to display the values